Makefile

```diff
@@ -4,12 +4,16 @@ isort = isort -rc src tests
 black = black src tests
 
 VERSION := $(shell python3 setup.py -V)
-RPMDIR := "$(shell pwd)/rpms-el7"
+RPMDIR-EL7 := "$(shell pwd)/rpms-el7"
 
 ifndef $(RELEASE)
 RELEASE := dev
 endif
 
+ifndef $(ISO_MAJOR_VERSION)
+ISO_MAJOR_VERSION := 5
+endif
```

Comment:
Wouldn't it be safer too ensure the end-user to always set this value? Except for an `OSS` build of course.